src/main/java/org/usfirst/frc/team1294/robot/OI.java

```diff
@@ -33,9 +35,50 @@
   private final Trigger dpadLeft;
   private final Trigger dpadUpLeft;
 
+  private final JoystickButton buttonA2;
```

Comment:
joystick2 & buttonA2 seems confusing to me..... perhaps naming the controllers driverController and gameMechController and the buttons driverControllerButtonA, gameMechControllerButtonA